operator/config/manager/manager.yaml

```diff
@@ -67,6 +67,8 @@ spec:
                   fieldPath: metadata.name
             - name: FUNCTION_RUNNER_IMAGE
               value: ghcr.io/keptn/functions-runtime:v0.7.1 # x-release-please-version
+            - name: PYTHON_RUNNER_IMAGE
+              value: docker.io/annadreal/python-runtime:test # TODO when we have the pipeline change this
```

Comment:
what tag do we use to release the image?  for this change to be valid we need for one to be released in our ghcr repo otherwise tests will all fail in main

Reply:
I propose we change this later when the image is there